src/app/models/AbstractModel.ts

```diff
@@ -74,6 +74,13 @@ export abstract class AbstractModel {
     return output;
   }
 
+  /**
+   * Convert model to string
+   */
+  public toString(): string {
+    return this.id.toString();
```

Comment:
This could be annoying; how does this look in the debug console?

Reply:
`toString()` does not affect the debug console output. Debug console displays the `toJSON()` method.